docs/sdk/web3-react/_category_.json

```
@@ -0,0 +1,5 @@
+{
+  "label": "Web3-react",
```

No caps, as it's the library name.

```suggestion
  "label": "web3-react",
```